src/ostorlab/cli/scanner/scanner.py

```diff
@@ -49,6 +68,7 @@ def scanner(
                 loop.close()
     if daemon is False and ctx.obj.get("api_key") is not None:
         loop = asyncio.get_event_loop()
+        loop.create_task(_start_periodic_persist_state(state_report=state_report))
```

Comment:
state reporter needs to get passed to subscribe_nats. Also please remove the _to_ from that method name.